content/en/docs/tasks/debug-application-cluster/debug-running-pod.md

```diff
@@ -0,0 +1,189 @@
+---
+reviewers:
+- verb
+- soltysh
+title: Debug Running Pods
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+This page explains how to debug Pods running (or crashing) on a Node.
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+* Your {{< glossary_tooltip text="Pod" term_id="pod" >}} should already be
+  scheduled and running. If your Pod is not yet running, start with [Troubleshoot
+  Applications](/docs/tasks/debug-application-cluster/debug-application/).
+* You should be familiar with {{< glossary_tooltip text="Ephemeral Containers"
```

Comment:
Is that always a prerequisite?

Reply:
No, that's a good point. Knowledge of ephemeral containers really only applies to one of these debugging methods. That section also links to the ephemeral container glossary entry, so I've just removed it from the prereqs. Happy to do something else if anyone has a better idea.